### ISSUE TITLE
PhaseTimeline

### ISSUE THREAD

**Issue by nlitwin** [appirio-tech/topcoder-app-r#14]
**Requirements**
- [ ] Please adhere to the code style specified in the readme.
- [ ] Use the `ProgressBar` components from `react-components`
- [ ] Make sure the component is rendered correctly on all screen sizes as specified in the designs

**Resources**
  - [Design specks](https://drive.google.com/open?id=0B1LbmKnex41fOWdBMHdGQU1jbTQ)

**Comment by nlitwin** [comment]
@vikasrohit Do you think we will be able to reuse the progress bar in react components for this?

![image](https://cloud.githubusercontent.com/assets/5829188/14296307/56a03fa4-fb2e-11e5-9082-9e63adab2a0a.png)

**Comment by vikasrohit** [comment]
Yes, ideally we should. That is what I was discussing with member who is working on this. I think the image you pasted, is possible even with the current implementation. However, what I was trying to achieve with that component is to break it down to two components one is ProgressBar and another is Timeline. ProgressBar would be the simple bar and Timeline would be using ProgressBar to show progress of each checkpoint. However, I am not able to think a way to implement this without using hacks (like negative margins). That is why I have put the refactoring of this component on hold until the challenge finishes. After that I would try to do that refactoring and once that is done, for all users of ProgressBar component would need to replace it with TimeLine component.

**Comment by nlitwin** [comment]
Ok sounds good! I think negative margins are ok :)
